src/index.ts

```diff
@@ -67,5 +67,11 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
 	CLIENT.on("guildDelete", guild => { resetConfigToDefault(guild.id); });
 
 	CLIENT.login(getApiToken());
+
+	process.on("SIGINT", () => {
```

Comment:
Do we need to care about other signals like `SIGTERM`, and `SIGQUIT` too?
https://stackoverflow.com/questions/46908853/process-onsigint-multiple-termination-signals
https://nodejs.org/api/process.html#process_signal_events

Reply:
I've never personally run into other signals, but I can add them if you want.

Reply:
That's fine, we'll notice if we close the bot and it doesn't say "destroyed client"